jvm/src/main/kotlin/org/bibletranslationtools/maui/jvm/io/HtmlWriter.kt

```diff
@@ -0,0 +1,12 @@
+package org.bibletranslationtools.maui.jvm.io
+
+import java.io.File
+
+class HtmlWriter {
+    fun write(filename:String, content:String) {
+        val file = File(filename)
+        file.printWriter().use {
+            it.println(content)
```

Comment:
are we going to leave it like this or add actual implementation later? Let's add a comment `// TODO` here explicitly.

Reply:
I don't know what you mean by "leave it like this" and insinuating that this isn't "actual implementation"

Reply:
I overlooked the writer part, thanks for the update